### ISSUE TITLE
Update Media Preview for REPL Links

### ISSUE THREAD

**Issue by shellscape** [rollup/rollupjs.org#190]
The current media preview for REPL links looks the same as if sharing https://rollupjs.org. We should give that some love so that REPL links are better described when sharing (especially on twitter, etc)

**Comment by lukastaegert** [comment]
To be honest, I have no idea how these previews are created, is this something we can control?

**Comment by lukastaegert** [comment]
Seems this is a good overview of what can be done and how: https://www.addtoany.com/ext/meta-tags

**Comment by shellscape** [comment]
Yeah, I'm familiar with how this is done. I just haven't been able to get to this yet.

**Comment by lukastaegert** [comment]
I assumed as much. This was more of a note if someone else wants to pick it up, there is this upcoming hackathon I am collecting small issues for, unless you definitely want to do it yourself?
